version.md

```diff
@@ -0,0 +1,94 @@
+# Versioning endpoints
+
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint represents a single version, matching a single item.
+
+Whether a version is accessible depends on `versioning.item.history.view.admin` configuration.  
+If this is set to true, the version can only be retrieved if the user is an admin of the related item
+
+## Create version
+
+TODO
+
+## Get single version
+
+**GET /api/versioning/version/<:versionId>**
+
+Provide version information for the version id.
+
+```json
+{
+  "id": "102",
+  "version": "2",
+  "type": "version",
+  "created": "2015-11-03T09:44:46.617",
+  "summary": "Fixing some typos in the abstract",
+  "_links": {
+    "versionhistory": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistory/10"
+    },
+    "self": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101"
+    },
+    "eperson": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/eperson"
+    },
+    "item": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/item"
+    },
+    "workspaceitem": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/workspaceitem"
+    },
+    "workflowitem": {
+      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/version/101/workflowitem"
```

Comment:
I'm wondering why `workspaceitem` and `workflowitem` need to be linked/embedded in the `version` endpoint.  Are they really used that often alongside the `version`?  Or can they just be retrieved (when necessary) via the linked `item`?

Reply:
There's no functionality to retrieve the workspaceitem or workflowitem via the linked item. So there's currently no alternative available.
An alternative would be to create a `/api/submission/workspaceitems/item/<:uuid>` endpoint which returns the workspace item matching the given item (not a search since it's always 0 or 1). The same would be possible for workflowitem. This may perhaps make more sense

Reply:
just to note that it should be a search method so `/api/submission/workspaceitems/search/item?uuid=<:uuid>`